concourse/scripts/generate-public-key-vars.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+
+gpg_public_keys = { "gpg_public_keys" => [] }
+
+public_key_ids = File.read('./.gpg-id')
+public_key_ids.each_line { |id|
+  # Assert key can be found locally
+  `gpg -k #{id}`
+  if $?.exitstatus != 0
+    puts "This key needs to be imported: #{id}"
+    puts """Try running
+    gpg --keyserver hkp://keyserver.ubuntu.com --recv #{id}
```

Comment:
I don't think we should bother specifying a keyserver here. This one doesn't match the one we use in the pipelines, and it's probably easiest to leave it using the gnupg default one.

Reply:
Done.